recipes/junifer/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "junifer" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/junifer-{{ version }}.tar.gz
+  sha256: 9244c954433e97858868b09d06adfdf20d238d96f1f7f5c276d712ab69e438a2
+
+build:
+  skip: true  # [py<38]
+  entry_points:
+    - junifer = junifer.api.cli:cli
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.0.0
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.8
+    - click >=8.1.3,<8.2
+    - numpy >=1.22,<1.24
+    - datalad >=0.15.4,<0.19
+    - pandas >=1.4.0,<1.6
+    - nibabel >=3.2.0,<4.1
+    - nilearn >=0.9.0,<=0.10.0
+    - sqlalchemy >=1.4.27,<=1.5.0
+    - pyyaml >=5.1.2,<7.0
+    - importlib-metadata  # [py<310]
```

Comment:
You can't use selectors in noarch recipes. In this case, it's better to just unconditionally depend on `importlib-metadata`:
```suggestion
```